features/tools.mdsd.ecoreworkflow.mwe2lib.xtext.feature/feature.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="tools.mdsd.ecoreworkflow.mwe2lib.xtext.feature"
+      label="MDSD.tools EcoreWorkflow MWE2 Library for Xtext"
+      version="0.1.0.qualifier"
+      plugin="org.palladiosimulator.branding">
+
+   <description url="http://www.example.com/description">
+      [Enter Feature Description here.]
+   </description>
+
+   <copyright url="http://www.example.com/copyright">
+      [Enter Copyright Description here.]
+   </copyright>
+
+   <license url="http://www.example.com/license">
+      [Enter License Description here.]
+   </license>
```

Comment:
We should think about creating a license feature for MDSD.tools features. But this is no blocker for this PR.

Reply:
👍